cli/src/cmd_net.rs

```diff
@@ -420,6 +434,275 @@ impl AuthenticatedCmd for CmdBgpFilter {
     }
 }
 
+#[derive(Parser, Debug, Clone)]
+#[command(verbatim_doc_comment)]
+#[command(name = "auth")]
+pub struct CmdBgpAuth {
+    /// Id of the rack to add the auth config to.
+    #[arg(long)]
+    rack: Uuid,
+
+    /// Switch to add the auth config to.
+    #[arg(long, value_enum)]
+    switch: Switch,
+
+    /// Port to add the auth config to.
+    #[arg(long, value_enum)]
+    port: Port,
+
+    /// Peer to apply allow list to.
+    #[arg(long)]
+    peer: IpAddr,
+
+    /// Authorization string.
+    #[clap(long)]
+    authstring: Option<String>,
+}
+
+#[async_trait]
+impl AuthenticatedCmd for CmdBgpAuth {
+    async fn run(&self, client: &Client) -> Result<()> {
+        let mut settings =
+            current_port_settings(client, &self.rack, &self.switch, &self.port).await?;
+        match settings.bgp_peers.get_mut(PHY0) {
+            None => return Err(anyhow::anyhow!("no BGP peers configured")),
+            Some(config) => {
+                let peer = config
+                    .peers
+                    .iter_mut()
+                    .find(|x| x.addr == self.peer)
+                    .ok_or(anyhow::anyhow!("specified peer does not exist"))?;
+
+                peer.md5_auth_key = self.authstring.clone();
+            }
+        }
+        client
+            .networking_switch_port_settings_create()
+            .body(settings)
+            .send()
+            .await?;
+        Ok(())
+    }
+}
+
+#[derive(Parser, Debug, Clone)]
+#[command(verbatim_doc_comment)]
+#[command(name = "pref")]
+pub struct CmdBgpLocalPref {
+    /// Id of the rack to add the route preference to.
```

Comment:
I might actually be removing the route preference, if this command allows it (since `--local-ref` is not a required argument according to the help output), so perhaps "set" is better than 'add"?

Similar for remaining help strings in this section.

Reply:
Changed add to set.